scripts/kubectl-kubestellar-remove-location

```diff
@@ -69,14 +69,26 @@ fi
 
 set -e
 
+echo "--- current directory is $PWD"
+cwsi=$(kubectl ws "${kubectl_flags[@]}" .)
+cwsi=${cwsi#*'"'}; cwsi=${cwsi%'"'*}
+echo "current ws is $cwsi"
+echo "imw is $imw"
 if [ "$imw" != "." ]
 then kubectl ws "${kubectl_flags[@]}" "$imw"
+     cwsi=$imw
+fi
+echo "current cwsi is $cwsi"
+if ! kubectl get apibinding "edge.kcp.io" &> /dev/null; then
+   kubectl kcp bind apiexport root:espw:edge.kcp.io
```

Comment:
Since this script is for ensuring the absence of an object, I do not think we want to ensure the presence of an APIBinding here.

Reply:
@MikeSpreitzer Is it possible to create synctargets.edge.kcp.io in a workspace without have created the APIBinding? And isn't possible to delete the APIBinding without deleting synctargets.edge.kcp.io? So then if you don't perform the binding you might have orphaned synctargets?

Reply:
It is impossible to create an object of a kind that is not defined.  I do not know whether it is possible to delete an APIBinding while objects of kinds defined by that APIBinding exist.  If someone did manage to remove the APIBinding while leaving such objects in storage, those objects would still not be accessible through to apiserver --- so that are effectively gone.  I do not think we need to worry here about provider bit tweezers for someone who manages to cause exceptional damage to their space.

Reply:
well I know it's possible to delete an APIBinding and still have orphaned objects in storage...because I did it while testing :--/ So the tweezer was for myself. But I'll remove it.